frappe/public/js/frappe/form/controls/link.js

```diff
@@ -472,12 +472,12 @@ frappe.ui.form.ControlLink = class ControlLink extends frappe.ui.form.ControlDat
 	}
 
 	get_filter_description(filters) {
-		let doctype = this.get_options();
+		const doctype = this.get_options();
 		let filter_array = [];
-		let meta = null;
 
+		// ensure meta is loaded (keeps existing behaviour)
 		frappe.model.with_doctype(doctype, () => {
-			meta = frappe.get_meta(doctype);
+			frappe.get_meta(doctype);
```

Comment:
This does nothing. Let's move the `frappe.model.with_doctype` to where it is needed and drop `frappe.get_meta`?